docs/how-to-setup-freecodecamp-mobile-app-locally.md

```diff
@@ -138,22 +138,10 @@ The default API keys and environment variables are stored in the file `sample.en
 # Populate it with the necessary API keys and secrets:
 ```
 
-<!-- tabs:start -->
-
-#### **macOS/Linux**
-
 ```console
 cp sample.env .env
 ```
 
-#### **Windows**
-
-```console
-copy sample.env .env
-```
-
-<!-- tabs:end -->
-
```

Comment:
This is not needed to remove, the mobile app support Windows development. Please remove only the opening and closing of the tabs, not the windows content here

Reply:
Sure, updating

Reply:
<img width="926" alt="image" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/79132440/c5c9f5e8-cfc5-4d6d-b533-3292322072b4">
 Is this okay so there's no conflict in headings?

Reply:
No, it's good that the headings are at the same level

Reply:
Cool, I have added them back based on what you have suggested. Can you check them again please?